test/workbox-routing/node/RegExpRoute.mjs

```diff
@@ -0,0 +1,89 @@
+import sinon from 'sinon';
+import {expect} from 'chai';
+
+import RegExpRoute from '../../../packages/workbox-routing/lib/RegExpRoute.mjs';
+import expectError from '../../../infra/testing/expectError.js';
+import {devOnly} from '../../../infra/testing/env-it.js';
+
+describe(`[workbox-routing] RegExpRoute`, function() {
+  const SAME_ORIGIN_URL = new URL('https://example.com');
+  const CROSS_ORIGIN_URL = new URL('https://cross-origin-example.com');
+  const PATH = '/test/path';
+  const HANDLER = {handle: () => {}};
+
+  const sandbox = sinon.sandbox.create();
+  sandbox.stub(global, 'location').value(SAME_ORIGIN_URL);
```

Comment:
I would probably add a beforeEach with sandbox restore and re-setting of the location stub - just to be safe (I suspect that if anyone adds a test to this class, this will get missed and cause confusion).